requirements.txt

```diff
@@ -1,4 +1,6 @@
 numpy==1.11.1
 pafy==0.5.2
+psycopg2==2.6.2
+sqlalchemy==1.0.14
 tornado==4.4.1
 youtube_dl==2016.8.17
```

Comment:
Actually... you're right. In this case I think it makes sense to allow it to be whatever.